src/shared/sam/debugger/csharpSamDebug.ts

```diff
@@ -145,6 +169,27 @@ async function _installDebugger(
     }
 }
 
+async function downloadInstallScript(debuggerPath: string): Promise<string> {
+    let installScriptUrl: string
+    if (os.platform() == 'win32') {
+        installScriptUrl = 'https://aka.ms/getvsdbgps1'
+    } else {
+        installScriptUrl = 'https://aka.ms/getvsdbgsh'
+    }
+
+    const installScriptFetcher = new HttpResourceFetcher(installScriptUrl)
+    const installScript = await installScriptFetcher.get()
+    if (!installScript) {
+        throw Error(`Failed to download ${installScriptUrl}`)
+    }
+
+    const installScriptPath = path.join(debuggerPath, 'installVsdbgScript')
+
+    await writeFile(installScriptPath, installScript, 'utf8')
+
+    return installScriptPath
+}
```

Comment:
This worked for me on Windows:
```suggestion
async function downloadInstallScript(debuggerPath: string): Promise<string> {
    let installScriptUrl: string
    let installScriptPath: string
    if (os.platform() == 'win32') {
        installScriptUrl = 'https://aka.ms/getvsdbgps1'
        installScriptPath = path.join(debuggerPath, 'installVsdbgScript.ps1')
    } else {
        installScriptUrl = 'https://aka.ms/getvsdbgsh'
        installScriptPath = path.join(debuggerPath, 'installVsdbgScript')
    }

    const installScriptFetcher = new HttpResourceFetcher(installScriptUrl)
    const installScript = await installScriptFetcher.get()
    if (!installScript) {
        throw Error(`Failed to download ${installScriptUrl}`)
    }

    await writeFile(installScriptPath, installScript, 'utf8')

    return installScriptPath
}
```

Reply:
Updated